pkg/build/controller/strategy/util.go

```diff
@@ -90,3 +65,49 @@ func setupBuildEnv(build *buildapi.Build, pod *kapi.Pod) error {
 	}
 	return nil
 }
+
+// setupDockerSecrets mounts Docker Registry secrets into Pod running the build,
+// allowing Docker to authenticate against private registries or Docker Hub.
+func setupDockerSecrets(pod *kapi.Pod, secretRef *kapi.ObjectReference) {
+	if secretRef == nil {
+		return
+	}
+
+	// TODO: Should namespace be required here?
```

Comment:
No, that should be error, I need to check that in validation.
On Mar 23, 2015 7:39 PM, "Jordan Liggitt" <notifications@github.com> wrote:

> In pkg/build/controller/strategy/util.go
> <https://github.com/openshift/origin/pull/1411#discussion_r26965923>:
>
> > @@ -90,3 +65,49 @@ func setupBuildEnv(build *buildapi.Build, pod *kapi.Pod) error {
> >  	}
> >  	return nil
> >  }
> > +
> > +// setupDockerSecrets mounts Docker Registry secrets into Pod running the build,
> > +// allowing Docker to authenticate against private registries or Docker Hub.
> > +func setupDockerSecrets(pod *kapi.Pod, secretRef *kapi.ObjectReference) {
> > +	if secretRef == nil {
> > +		return
> > +	}
> > +
> > +	// TODO: Should namespace be required here?
>
> should a namespace other than the pod's namespace even be allowed?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/1411/files#r26965923>.
>